packages/cfg/test/findAncestor.test.ts

```diff
@@ -1,5 +1,5 @@
-import { CFG, NodeType } from "../../../src";
 import * as chai from "chai";
+import { CFG, NodeType } from "@syntest/cfg";
```

Comment:
Why do we import the npm package in the local test? This would prevent us from testing new changes. This should most likely need to point to the `index.ts`